test-runner/golint.sh

```diff
@@ -13,12 +13,15 @@ if [ -n "$1" ]; then
     MODULE_DIR=$1
 fi
 
-pushd ${MODULE_DIR}
+mkdir lint
+pushd lint
+go mod init example.com/lint
+go get -d golang.org/x/lint/golint
```

Comment:
nit, from the docs 'In Go 1.18, -d will always be enabled', so we could omit that.

Reply:
This still runs on our OSPdO 1.2.x branch though so I think not yet right?

Reply:
right forgot about 1.2.x .... so -d is good